quarkus/tests/integration/src/main/java/org/keycloak/it/junit5/extension/WithDatabase.java

```diff
@@ -33,8 +33,11 @@
 
     /**
      * The database name as per database aliases.
-     *
-     * @return the database alias
      */
     String alias();
+
+    /**
+     * If the {@code db} option must be unset after the running the build command.
+     */
+    boolean removeBuildOptionsAfterBuild() default false;
```

Comment:
I believe that this should default to `true`

Reply:
Why?

Reply:
Ok

Reply:
Actually, I would prefer to not do it now because we need to look at other tests and see how this option changes their expectations.

We do have tests covering this area but for other situations and configuration options. It should be fine to keep it `false` and enabled only for this test.